ppr-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ppr-ui",
-  "version": "0.4.0",
+  "version": "0.4.1",
```

Comment:
Should be 0.4.2

Reply:
You bet, as with all the other pending PR's. Will update once the others are updated/in 👍